qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -498,6 +646,7 @@ def generate_hamiltonian(
     n_active_orbitals=None,
     mapping="jordan_wigner",
     outpath=".",
+    wires=None,
 ):  # pylint:disable=too-many-arguments
```

Comment:
After we merge some of the pending PRs we might need to solve few conflicts, I guess.